packages/flutter/lib/src/gestures/events.dart

```diff
@@ -292,6 +292,10 @@ abstract class PointerEvent {
 
   /// Returns a complete textual description of this event.
   String toStringFull() {
+    String extraDescription = _extraDescription();
+    if (extraDescription.isNotEmpty) {
+      extraDescription = ', ' + extraDescription;
+    }
```

Comment:
we really should just make this implement Diagnosticable. cc @jacob314

Reply:
Attempting to do so gives me:
```
'PointerEvent.toString' ('() → String') isn't a valid override of 'Diagnosticable.toString' ('({minLevel: DiagnosticLevel}) → String').
```
Obviously I could fix that by adding the parameter to PointerEvent, but that's an API breaking change that would impact anyone who has subclassed PointerEvent and overridden toString. Suggestions?

Reply:
Don't worry about it, nobody will have done that.

Reply:
Sent #27904 to convert the existing logging prior to adding more logging here.